DataFormats/L1TMuon/interface/L1TMuonSubsystems.h

```diff
@@ -0,0 +1,8 @@
+#ifndef DataFormats_L1TMuon_L1TMuonSubsystems_hh
+#define DataFormats_L1TMuon_L1TMuonSubsystems_hh
+
+namespace L1TMuon {
```

Comment:
Would a `enum class` be better since this is presently violating the CMS naming rules for namespaces.

Reply:
this namespace is extensively already...

Reply:
but of course its easy to change if we don't mind it being different from the rest of the L1TMuon code.